src/raygun.loader.js

```diff
@@ -24,7 +24,7 @@
   errorQueue = window[window['RaygunObject']].q;
   var rg = Raygun;
 
-  var delayedExecutionFunctions = ['trackEvent', 'send', 'recordBreadcrumb'];
+  var delayedExecutionFunctions = ['trackEvent', 'send', 'recordBreadcrumb','captureMissineRequests'];
```

Comment:
can you tell me more about this change?

Reply:
Well firstly I just noticed that its spelt wrong 😅 This is mostly unrelated to the primary issue that the PR is trying to solve. However, the `captureMissingRequests` function uses the offset var which I had modified, so when testing I wanted to verify that I hadn't broken it. This is when I found that in [this function](https://github.com/MindscapeHQ/raygun4js/blob/e5cbbbf1b7166bc1a418b6d8566cca0cda540db8/src/raygun.js#L388) `_rum` seems to always be undefined which results in `captureMissingRequests` never being set. Adding it into the `delayedExecutionFunctions` means that it is run after the `_rum` object is created. Now that I think about it this might result in the first page load missing any of these requests as the property might be set after the occur? 🤔

Reply:
hmm, possibly something to test - it makes sense to delay this as it's calling the _rum.captureMissingRequests and if _rum is always null then it wouldn't have been doing anything anyway. I do see a null check inside the `captureMissingRequests`, I wonder if this was to mitigate this